sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -63,7 +63,7 @@ def update(self, *args: Any, **kwargs: Any) -> None:
     def keys(self) -> List[str]:
```

Comment:
Are you getting any sample errors relate to AMQPMessage annotation/delivery_annotations/message_properties types?

Reply:
I am not.